soap/LOG.md

```
@@ -0,0 +1,19 @@
+# لاگ اپ SOAP
+
+این فایل برای ثبت مشکلات/تناقض‌ها و وضعیت پیشرفت استفاده می‌شود.
+
+## تناقض‌ها / بلوکرها
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)